src/client/user/User.js

```diff
@@ -51,14 +51,17 @@ export default class User extends React.Component {
   };
 
   componentDidMount() {
-    if (!this.props.user.name) {
-      this.props.getAccountWithFollowingCount({ name: this.props.match.params.name });
+    if (!this.props.user.name && this.props.authenticatedUser.name) {
+      this.props.getAccountWithFollowingCount({ name: this.props.match.params.name,
```

Comment:
It doesn't look to good on the same line, I would put this object properties on the new line each.